launch/planning_scene/planning_scene_visualization.launch

```diff
@@ -1,18 +1,14 @@
 <?xml version="1.0"?>
 <launch>
 
-  <arg name="world_config" default="ai_day" doc="objects arrangement, options: moelk_tables, cic_tables, truck_assembly, ai_day" />
-  <arg name="yaml_path_to_read" default="$(find tables_demo_bringup)/config/$(arg world_config)_planning_scene.yaml" />
-  <arg name="yaml_path_to_write" default="$(arg yaml_path_to_read)" />
+  <arg name="yaml_path"/>
```

Comment:
How about using an example config file from [config/examples](https://github.com/DFKI-NI/grasplan/tree/noetic/config/examples) as the default, as suggested by @oscar-lima in https://github.com/DFKI-NI/grasplan/issues/22#issuecomment-2047370288?